scripts/kind-e2e/lib_operator_deploy_subm.sh

```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
 # This should only be sourced
-if [ "${0##*/}" = "lib_operator_deploy_subm.sh" ]; then
+if [ `basename "$0"` = "lib_operator_deploy_subm.sh" ]; then
```

Comment:
Why the switch to `basename`?

Reply:
Because @mkolesnik asked me to :)

Reply:
Please restore the original, it’s not a bashism and is perfectly valid POSIX shell.

Reply:
OK so it's a bit of obscure shellism :)

I don't have a strong opinion about it so whatever @skitt prefers is OK by me.

Reply:
Done in the latest